src/Impostor.Api/Events/Game/Player/IPlayerVotedEvent.cs

```diff
@@ -0,0 +1,12 @@
+using Impostor.Api.Net.Inner.Objects;
+
+namespace Impostor.Api.Events.Player
+{
+    public interface IPlayerVotedEvent : IPlayerEvent
+    {
+        /// <summary>
+        ///     Get the player he voted for.
+        /// </summary>
+        IInnerPlayerControl? VotedFor { get; }
```

Comment:
This is not warning my IDE that VoteFor might be null and allows me to compile

Reply:
check example pls https://github.com/Impostor/Impostor/pull/181/commits/6d02e86856ddadd9545f302bd8f794c0c17d5d3f

Reply:
add `#nullable enable`

Reply:
My bad - I forgot visual studio doesn't enforce nullable `?` when typing nullables